crates/bevy_render/src/texture/image_loader.rs

```diff
@@ -38,29 +38,44 @@ const FILE_EXTENSIONS: &[&str] = &[
     "webp",
 ];
 
-impl AssetLoader for ImageTextureLoader {
+#[derive(Serialize, Deserialize, Default)]
+pub enum ImageFormatSetting {
```

Comment:
I need a way to set the image's `sampler_descriptor` when loading the FBX. Could this be added to `ImageFormatSetting`?

Reply:
Definitely! The current settings were just the absolute minimum required to make Basis Universal encoding work.

Reply:
On second thought. I know that `tracing` has difficulties across async boundaries, so it might be an issue. But I'll check it out if you add it.